Auth0Tests/AuthenticationSpec.swift

```diff
@@ -841,6 +979,143 @@ class AuthenticationSpec: QuickSpec {
                     }
                 }
             }
+            
+            context("passwordless login") {
```

Comment:
**Nit:** Testing all of this twice seems duplicative. Didn't the last batch cover the private method?

Reply:
Since the private method is an implementation detail and the subject under test (an instance of a type conforming to the `Authentication` protocol) is considered a black box, I think I should be testing separately the login with email and sms functionality, regardless if it's implemented using the same method or not.